client/python/README.md

```diff
@@ -56,13 +56,12 @@ client = grc.Client(src="btd372-js72hd.gradio.app")
 
 The simplest way to make a prediction is simply to call the `.predict()` function with the appropriate arguments and then immediately calling `.result()`, like this:
 
-
 ```python
 import gradio_client as grc
 
 client = grc.Client(space="abidlabs/en2fr")
 
-client.predict("Hello").result()
+client.make_predict("Hello").result()
```

Comment:
Could we keep the name of the function `.predict()` for a nicer user experience?

Reply:
The name is still predict - this is the auto refactoring feature gone wrong. Will fix!